src/encoded/commands/deploy.py

```diff
@@ -3,6 +3,10 @@
 import re
 import subprocess
 import sys
+import datetime
+import base64
+import pdb
+from pprint import pprint as pp
```

Comment:
Need base64 so the encoding can be set and returned correctly. Date and time is used for checking todays prices. The others are gone.